jdi-light-mobile-examples/src/main/java/selenium/site/data/ContactInfo.java

```diff
@@ -0,0 +1,10 @@
+package selenium.site.data;
+
+import com.epam.jdi.tools.DataClass;
+
+public class ContactInfo extends DataClass<ContactInfo> {
+    public String name, lastName, position,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)